playbooks/common/openshift-cluster/upgrades/rpm_upgrade.yml

```diff
@@ -5,7 +5,3 @@
 - name: Ensure python-yaml present for config upgrade
   action: "{{ ansible_pkg_mgr }} name=PyYAML state=present"
   when: not openshift.common.is_atomic | bool
```

Comment:
Should the Upgrade packages task above be updated to notify a handler? Or are we forcing a node restart later in the process.